src/types/ErrorMessages.ts

```diff
@@ -0,0 +1,8 @@
+export enum ErrorMessages {
+  NONE = '',
+  ONLOAD = 'Unable to load todos',
+  ONTITLE = 'Title can\'t be empty',
+  ONADD = 'Unable to add todo',
+  ONDELETE = 'Unable to delete todo',
+  ONUPDATE = 'Unable to update a todo',
```

Comment:
Usually, we do not store all the text inside the enum, enum is used for a short list of values
Display different text inside component depending on this enum